org-code-javabuilder/neighborhood/src/main/java/org/code/neighborhood/Painter.java

```diff
@@ -293,6 +305,13 @@ private HashMap<String, String> getSignalDetails() {
     return details;
   }
 
+  private HashMap<String, String> getSignalDetailsForBooleanMethod(boolean result) {
+    HashMap<String, String> details = this.getSignalDetails();
+    String resultString = result ? "true" : "false";
```

Comment:
nit: I think you could just do `String.valueOf(result)` here?

Reply:
yep, updated!